xml/System.Net/ServicePointManager.xml

```diff
@@ -824,9 +824,13 @@ For versions of the .NET Framework through the .NET Framework 4.6.2, no default
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- An application can set the <xref:System.Net.ServicePointManager.ServerCertificateValidationCallback%2A> property to a method to use for custom validation by the client of the server certificate. When doing custom validation, the `sender` parameter passed to the <xref:System.Net.Security.RemoteCertificateValidationCallback> can be a host string name or an object derived from <xref:System.Net.WebRequest> (<xref:System.Net.HttpWebRequest>, for example) depending on the <xref:System.Net.ServicePointManager.CertificatePolicy%2A> property.  
+An application can set the <xref:System.Net.ServicePointManager.ServerCertificateValidationCallback%2A> property to a method to use for custom validation by the client of the server certificate. When doing custom validation, the `sender` parameter passed to the <xref:System.Net.Security.RemoteCertificateValidationCallback> can be a host string name or an object derived from <xref:System.Net.WebRequest> (<xref:System.Net.HttpWebRequest>, for example) depending on the <xref:System.Net.ServicePointManager.CertificatePolicy%2A> property.  
   
- When custom validation is not used, the certificate name is compared with host name used to create the request. For example, if <xref:System.Net.WebRequest.Create%28System.String%29> was passed a parameter of "https://www.contoso.com/default.hmtl", the default behavior is for the client to check the certificate against www.contoso.com.  
+When custom validation is not used, the certificate name is compared with host name used to create the request. For example, if <xref:System.Net.WebRequest.Create%28System.String%29> was passed a parameter of "https://www.contoso.com/default.hmtl", the default behavior is for the client to check the certificate against www.contoso.com.  
+
+Despite being a multicast delegate, only the value returned from the last-executed event handler is considered authoritative. 
+In other words, you can attach multiple delegates and they all get a callback from <xref:System.Net.ServicePointManager.ServerCertificateValidationCallback%2A>. 
+Each callback would return a value that indicates whether the certificate will be accepted or not, however only the value from the last delegate will be respected.
```

Comment:
semicolon after "accepted or not"
comma after "however"
It's clearer to phrase this in the present tense:
- callback would return a value --> callback returns a value
- certificate will be accepted --> certificate is accepted
- last delegate will be respected --> last delegate is respected